### PR TITLE
Add Denver recommendations

### PR DESCRIPTION
**Pull request by doppeldimmykumont**
This PR closes issue #33.
## Adding restaurants and activities

-What types of food do you enjoy?
-Do you like the great outdoors?

@githubteacher

### CODE REVIEW

File: doppeldimmykumont/doppeldimmykumont-denver.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Denver :tada:
+
+#### Restaurants
```
Comment (ghost) [inline doppeldimmykumont/doppeldimmykumont-denver.md:3]: @doppelDimmykumont  what restaurant would you recommend for Mexican food? :smiley: :meat_on_bone:

Comment (doppeldimmykumont) [inline doppeldimmykumont/doppeldimmykumont-denver.md:3]: @moyomendoza Definitely Los Carboncitos! Best tacos in town.

Comment (jdavis1998) [inline doppeldimmykumont/doppeldimmykumont-denver.md:3]: Which location?

Comment (doppeldimmykumont) [inline doppeldimmykumont/doppeldimmykumont-denver.md:3]: @jdavis1998 Los Carboncitos on 38th!

Comment (ghost) [inline doppeldimmykumont/doppeldimmykumont-denver.md:3]: hehe,  it also has a funny name,  sounds like grilled meat,  I just got hungry

**Comment by doppeldimmykumont** [comment]
:shipit:

**Pull request merged**